bash_functions.sh

```diff
@@ -22,14 +22,20 @@ prepare_configs() {
     # Re-apply perms from basic-install over any volume mounts that may be present (or not)
     # Also  similar to preflights for FTL https://github.com/pi-hole/pi-hole/blob/master/advanced/Templates/pihole-FTL.service
     chown pihole:root /etc/lighttpd
-    chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" "${regexFile}"
+    chown pihole:pihole "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" "/var/log/pihole" 
     chmod 644 "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf"
     # not sure why pihole:pihole user/group write perms are not enough for web to write...dirty fix:
-    chmod 777 "${regexFile}"
+    if [[ -n ${regexFile} && -e ${regexFile} ]]; then 
+      chown pihole:pihole "${regexFile}"
+      chmod 777 "${regexFile}"
```

Comment:
Can you verify that `rwx` is needed for this file still?

Reply:
I'm not sure i couldn't find anywhere in the script that the code was used.  i didn't want to remove it i just insulated it from spewing errors put in the logs.

Reply:
see my comment on Issue #660: that variable is never set so I would think the stray references should just be removed.